webapp/src/main/resources/public/js/stores/workbench/ViewModeStore.js

```diff
@@ -0,0 +1,28 @@
+import alt from "../../alt";
+import ViewModeActions from "../../actions/workbench/ViewModeActions";
+import { readCookie, writeCookie } from "../../utils/Cookie";
```

Comment:
can we store this in local storage instead since it is client side only configuration?

Reply:
Do you mean with `local storage` inside the DB in the server?

If you mean just saved in JS (it is gone if you refresh the page), then I would like to keep it a little more persistent so that I don't have to change it back every time.

Reply:
browser [local storage](https://developer.mozilla.org/en-US/docs/Web/API/Window/localStorage), that should support browser restart, etc

Reply:
Ah, I wasn't aware of that. I have changed it to the local storage now.